AgileCoffeeShop/Test/CoffeeShopTests.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using NUnit.Framework;
+
+namespace AgileCoffeeShop.Test
+{
+    [TestFixture]
+    class CoffeeShopTests
+    {
+        [Test]
+        public void TestCalculateDistances()
```

Comment:
Why did you decide to write only 1 test and this certain test?

Reply:
I decided to write only this test because it is the main functionality and I'm not very familiar with the tests. I learned them recently and I still have to learn.